core/types/transaction_signing.go

```diff
@@ -242,7 +246,7 @@ func (fs FrontierSigner) SignatureValues(tx InternalTransaction, sig []byte) (r,
 func (fs FrontierSigner) Hash(tx InternalTransaction) common.Hash {
 	return hash.FromRLP([]interface{}{
 		tx.Nonce(),
-		tx.GasPrice(),
+		tx.RawGasPrice(),
```

Comment:
what's the reason to change GasPrice to RawGasPrice？

Reply:
good catch! as far as I remember, some harmony tests were failing because the new `GasPrice()` implementation calls `IsAA()` which assumes that transaction is signed, whereas it was not the case in tests. This should be symantically equivalent to what upstream implementation is doing (since they are not calling `GasPrice()`, they do `tx.data.Price` which equals to `RawGasPrice()`).